src/components/PackShipDatePicker.jsx

```diff
@@ -22,7 +22,7 @@ const PackShipDatePicker = ({
         onChange={onChange}
         renderInput={(params) => <TextField {...params} />}
       />
-      {displayDateHelper && value ? (
+      {displayDateHelper ? (
```

Comment:
Should we also make the box show up as red?

Reply:
Made a change. Let me know what you think.

Reply:
I think the grey makes it look disabled now, and I just found a bug where if you hit next with nothing in it, type in 1 number and then hit next again, it will go through with it. Also if I hit next and type a number after the error, it immediately loses focus and I can't type again.